manim/mobject/three_dimensions.py

```diff
@@ -59,10 +57,8 @@ def construct(self):
     def __init__(
         self,
         func,
-        u_min=0,
-        u_max=1,
-        v_min=0,
-        v_max=1,
+        u_range=[0, 1],
+        v_range=[0, 1],
         resolution=32,
         surface_piece_config={},
         fill_color=BLUE_D,
```

Comment:
Currently, these are breaking changes since `u_min/u_max` would error out, which I'm generally okay with but would like to avoid when unecessary. Since we have @deprecated_params now, I think we should try to use it here to prevent old code from breaking. I used it in another PR to deprecate `size`, [see here](https://github.com/ManimCommunity/manim/blob/5320a6e8f12b0145582dfd9a42b03e617bf9c1a9/manim/mobject/svg/text_mobject.py#L396-L433). You could try to follow a similar approach.

Reply:
I understand. I'll try and do that. Do I add the deprecation for all three d mobjects which had the min max notation or just the parametric surface? @hydrobeam

Reply:
just the parametric surface since the mobjects inherit from it

Reply:
I'll do all